playbooks/common/openshift-master/config.yml

```diff
@@ -206,6 +206,7 @@
     when: openshift_use_nuage | default(false) | bool
   - role: calico_master
     when: openshift_use_calico | default(false) | bool
+  - role: tuned
```

Comment:
We probably don't need this on the master playbook because the masters get the node configuration as well, I believe.